ood-portal-generator/spec/ood_portal_generator_view_spec.rb

```diff
@@ -15,7 +15,7 @@
       example_config_opts -= %w(dex) 
       
       # delete inst vars that are not actual options in the example file
-      config_opts -= %w(protocol oidc_redirect_uri oidc_crypto_passphrase dex_http_port)
+      config_opts -= %w(protocol rails_config_hosts oidc_redirect_uri oidc_crypto_passphrase dex_http_port)
```

Comment:
We can kick this to another ticket (because it requires a lot more updates) but we should actually put this in the shared example.

Reply:
Nevermind, I'm actually now realizing that this is `server_aliases` that folks are interested in, not this internal attribute.